cirq/google/calibration/workflow.py

```diff
@@ -390,21 +535,83 @@ def run_floquet_characterization_for_circuit(
+    return circuit_calibration, results
+
+
+def run_zeta_chi_gamma_calibration_for_moments(
+    circuit: Circuit,
+    engine: Union[Engine, PhasedFSimEngineSimulator],
+    processor_id: Optional[str] = None,
+    gate_set: Optional[SerializableGateSet] = None,
+    options: FloquetPhasedFSimCalibrationOptions = ZETA_GAMMA_FLOQUET_PHASED_FSIM_CHARACTERIZATION,
+    gates_translator: Callable[[Gate], Optional[FSimGate]] = try_convert_sqrt_iswap_to_fsim,
+    merge_subsets: bool = True,
+    max_layers_per_request: int = 1,
+    progress_func: Optional[Callable[[int, int], None]] = None,
+) -> Tuple[CircuitCalibration, List[PhasedFSimCalibrationResult]]:
+    """Compensates circuit against errors in zeta, chi and gamma angles by running calibrations on
+    the engine.
+
+    Args:
+        circuit: Circuit to characterize and calibrate.
+        engine: cirq.google.Engine or cirq.google.PhasedFSimEngineSimulator object used for running
+            the calibrations. When cirq.google.Engine then processor_id and gate_set arguments must
+            be provided as well.
+        processor_id: processor_id passed to engine.run_calibrations method. Can be None when
+            cirq.google.PhasedFSimEngineSimulator is used as an engine.
+        gate_set: Gate set to use for characterization request. Can be None when
+            cirq.google.PhasedFSimEngineSimulator is used as an engine.
+        options: Options that are applied to each characterized gate within a moment. Defaults
+            to all_except_for_chi_options which is the broadest currently supported choice.
+        gates_translator: Function that translates a gate to a supported FSimGate which will undergo
+            characterization. Defaults to sqrt_iswap_gates_translator.
+        merge_subsets: Whether to merge moments that can be characterized at the same time
+            together.
+        max_layers_per_request: Maximum number of calibration requests issued to cirq.Engine at a
+            single time. Defaults to 1.
+        progress_func: Optional callback function that might be used to report the calibration
+            progress. The callback is called with two integers, the first one being a number of
+            layers already calibrated and the second one the total number of layers to calibrate.
+
+    Returns:
+        Tuple of:
+          - Calibrated circuit together with its calibration metadata in CircuitCalibration object.
+            The calibrated circuit has single-qubit Z gates added which compensates for the true
+            gates imperfections.
+          - List of characterizations that were triggered in order to calibrate the circuit.
+    """
+    circuit_calibration, requests = make_floquet_request_for_circuit(
+        circuit, options, gates_translator, merge_subsets=merge_subsets
+    )
+    characterizations = run_characterizations(
+        requests,
+        engine,
+        processor_id,
+        gate_set,
```

Comment:
since the formatter is putting these all on their own lines, you could make them all kwargs to 1) drill home the point that you're just forwarding the values 2) prevent accidental reordering either here or changing the `run_characterizations` signature

Reply:
Done.